Python/pywarpx/picmi.py

```diff
@@ -714,6 +714,39 @@ def initialize_inputs(self):
         pywarpx.warpx.mirror_z_npoints.append(self.number_of_cells)
 
 
+class MCCCollisions(object):
+    """Custom class to handle setup of MCC collisions in WarpX. If collision
+    initialization is added to picmistandard this can be changed to inherit
+    that functionality."""
+
+    def __init__(self, name, species, background_density,
+                 background_temperature, scattering_processes,
+                 background_mass=None):
+        self.name = name
+        self.species = species
+        self.background_density = background_density
+        self.background_temperature = background_temperature
+        self.background_mass = background_mass
+        self.scattering_processes = scattering_processes
+
+    def initialize_inputs(self):
+        collision = pywarpx.Collisions.newcollision(self.name)
+        collision.type = 'background_mcc'
+        collision.species = self.species.name
+        collision.background_density = self.background_density
+        collision.background_temperature = self.background_temperature
+        collision.background_mass = self.background_mass
+
+        collision.scattering_processes = self.scattering_processes.keys()
+        for process, kw in self.scattering_processes.items():
+            for key, val in kw.items():
+                if key == 'species':
+                    val = val.name
+                collision.add_new_attr(process+'_'+key, val)
+
+        pywarpx.Collisions.collisions_list.append(collision)
+
```

Comment:
Why does the newcollision function append to "collisions_list" when the "collision" also gets appended in the last line of 
"initialize_inputs" in "MCCCollisions"?